### PR TITLE
adding changes to request PUG chem API; added DTO objects to handle u…

### PR DESCRIPTION
**Pull request by dnguyen000**
…ser input

adding changes to application.properties

### CODE REVIEW

**Review by atvick21 — APPROVED** [review]

**Review by f947 — APPROVED** [review]

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -12,4 +12,10 @@ spring.data.mongodb.host=localhost
 # App Config
 #server.port=8102
 spring.application.name=Chemistry
-#server.servlet.context-path=/user
\ No newline at end of file
+#server.servlet.context-path=/user
+
+pubchem.host=pubchem.ncbi.nlm.nih.gov
```
Comment (calebTree) [inline src/main/resources/application.properties:17]: These variables should be setup with the names the API uses: [from the tutorial](https://pubchemdocs.ncbi.nlm.nih.gov/pug-rest-tutorial$_Toc458584410).

Comment (calebTree) [inline src/main/resources/application.properties:17]: These variables should be setup with the names the API uses: [from the tutorial](https://pubchemdocs.ncbi.nlm.nih.gov/pug-rest-tutorial$_Toc458584410).

**Comment by calebTree** [comment]
I don't know why this isn't allowing me to approve this PR, I do approve it however.

**Comment by f947** [comment]
If you refresh your web browser page, then you will be able to approve it.
That happened to me.

**Comment by f947** [comment]
Or you can do sign out and sign in, that might work too.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -12,4 +12,11 @@ spring.data.mongodb.host=localhost
...
 #server.port=8102
 spring.application.name=Chemistry
-#server.servlet.context-path=/user
\ No newline at end of file
+#server.servlet.context-path=/user
+
+pubchem.prolog=https://pubchem.ncbi.nlm.nih.gov/rest/pug
+pubchem.input=/compound/fastformula/
+pubchem.operation=/property/MolecularFormula,MolecularWeight,Title
+pubchem.output=/JSON
```
Comment (calebTree) [inline src/main/resources/application.properties:20]: These variables should be setup with the names the API uses: [from the tutorial](https://pubchemdocs.ncbi.nlm.nih.gov/pug-rest-tutorial$_Toc458584410).

- I'll push a commit to this branch that has that update.

**Review by calebTree — APPROVED** [review]

**Comment by calebTree** [comment]
Thanks @f947. Apparently, when in-line comments are made, the review button in the top right of the files tab needs to be clicked on and a review option selected.

**Pull request merged**